test/EndToEndTests/Tests/Client/AsynchronousTests/AsyncMethodTests.cs

```diff
@@ -105,7 +105,7 @@ public async Task SaveChangesTest()
             customers.Add(c2);
 
             var dataServiceResponse = await context.SaveChangesAsync(SaveChangesOptions.BatchWithIndependentOperations | SaveChangesOptions.UseRelativeUri);
-            Assert.Equal((dataServiceResponse.First() as ChangeOperationResponse).StatusCode, 201);
+            Assert.Equal(201, (dataServiceResponse.First() as ChangeOperationResponse).StatusCode);

```

Comment:
While you're here, you might want to use the `HttpStatusCode` Enum, for readability
https://docs.microsoft.com/en-us/dotnet/api/system.net.httpstatuscode